src/transformers/trainer.py

```diff
@@ -1140,7 +1141,28 @@ def call_model_init(self, trial=None):
 
         return model
 
-    def _wrap_model(self, model, training=True):
+    def torch_jit_model(self, model, training=False, dataloader=None):
+        if not training:
+            jit_inputs = []
+            example_batch = list(dataloader)[0]
+            for _, key in enumerate(example_batch):
```

Comment:
```suggestion
            for key in example_batch:
```
Again, why enumerate when the index is not used?